### ISSUE TITLE
triplet loss or cross entropy loss?

### ISSUE THREAD

**Issue by srika91** [nwojke/cosine_metric_learning#21]
I have attached my training and validation graphs.am not able to reduce the cross entropy loss beyond 2.4 and the gap between training and validation accuracy is 30% most of the time.I understand this is overfitting but whatever I do am not able to reduce this overfitting.Attached few images for reference.My doubt is in the paper only the triplet  loss is shown not the cross entropy loss like it is shown here.

![graph1](https://user-images.githubusercontent.com/7530485/45759863-803cc900-bc46-11e8-821d-38d31731fb67.png)
![graph2](https://user-images.githubusercontent.com/7530485/45759859-7fa43280-bc46-11e8-8133-81aff29e3eb8.png)
![graph3](https://user-images.githubusercontent.com/7530485/45759857-7f0b9c00-bc46-11e8-92b6-e078de55a7fb.png)
![graph5](https://user-images.githubusercontent.com/7530485/45759854-7e730580-bc46-11e8-9cc4-8dca47f917ea.png)

Anyhelp will be great

**Comment by srika91** [comment]
@nwojke  any thoughts on this.am training for VeRi dataset

**Comment by ss199302** [comment]
@srika91  Will you use triple loss effect?

**Comment by nwojke** [comment]
Hi srika91, sorry for the late reply. Regarding overfitting, note that the identities in the training and validation split are not the same. Therefore, you are likely not going to get Precision@1 similar to training accuracy (training accuracy is the accuracy of the classifier, Precision@1is the re-identification accuracy on unseen identities).

I have re-run the training scrip on VeRi a few days ago and this is how training statistics look like for me:

![export_training](https://user-images.githubusercontent.com/25549077/57070752-a8120600-6cd8-11e9-8669-164f86eecac4.png)

This is Precision@1 no the validation split (after around 100k steps):

![export_validation](https://user-images.githubusercontent.com/25549077/57070779-b829e580-6cd8-11e9-8f96-166edc78c80f.png)

Looks quite similar.

**Comment by nwojke** [comment]
I am closing this issue due to inactivity. Re-open if the problem is not solved.

**Comment by cekcoco** [comment]
hi ,I train my data by using the code of the author,but I can not get the precision_1 in my tensorboard.could you give me some advice?
